### PR TITLE
add questions

### PR DESCRIPTION
**Pull request by AL743-ni82dy**

### CODE REVIEW

File: HIGHLIGHTS.md [hunk 1]
```diff
@@ -4,6 +4,15 @@
 
 What highlights can you come up with to share with the class?
 
+- codecademy - provides a bit more code defintion, Jessica
+- w3schools.com - helps as a reference in overall Javascript, Alvin
+- css-tricks.com - helps with more css, a good walk through, Rob
```
Comment (rock43053) [inline HIGHLIGHTS.md:9]: LGTM

File: HIGHLIGHTS.md [hunk 2]
```diff
@@ -4,6 +4,15 @@
...
+- Javascript for Dummies book - a good beginner's guide, Alejandro
+
+- codewithmosh.com - his free videos on Youtube are older, pay website is more recent
+- Youtube - in general, can be searched for videos on various subjects
+
+
```
Comment (ghost) [inline HIGHLIGHTS.md:15]: There are all amazing resources!!!! I will copy and pasta them into the `#ramdom` channel

Thank you!

File: QUESTIONS.md [hunk 3]
```diff
@@ -1,5 +1,19 @@
...
+- How often is new code created vs just updating existing code in a typical work environment
+- What is a relational database?
+
+
 What questions can you come up with?
 
 ## Cohort Questions
+
+- Can we see the syllabus, again?
+- When is the website portfolio due
```
Comment (Emeralds99) [inline QUESTIONS.md:19]: +1

**Review by Emeralds99 — APPROVED** [review]
🦩 🐦

File: QUESTIONS.md [hunk 4]
```diff
@@ -1,5 +1,19 @@
 # Questions about Code, Node, Express, React
 
+- Can you talk about file IO and elaborate on it
+- Can you teach us more on React and what its functions are
+- What is the difference between Angular and React
+- How often is new code created vs just updating existing code in a typical work environment
+- What is a relational database?
```
Comment (hollisjamison) [inline QUESTIONS.md]: Short answer:
Relational works via tables where properties are RELATED by certain key fields

Nonrelational works via collections where "documents' (basically javascript obeects) with ALL properties are stored as one big thing.

Long answer: watch this incredible video: https://www.youtube.com/watch?v=ZS_kXvOeQ5Y

**Pull request merged**